packages/cache-memcached/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@medusajs/cache-memcached",
+  "version": "1.0.0",
+  "description": "Memcached Cache Module for Medusa",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/medusajs/medusa",
+    "directory": "packages/cache-memcached"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "author": "Medusa",
+  "license": "MIT",
+  "devDependencies": {
+    "@medusajs/medusa": "*",
```

suggestion(non-blocking): 
```suggestion
    "@medusajs/medusa": "workspace:*",
```